sct.py

```diff
@@ -11,6 +11,7 @@
 
 from sdcm.results_analyze import PerformanceResultsAnalyzer
 from sdcm.sct_config import SCTConfiguration
+from sdcm.utils.cloud_monitor.cloud_monitor import cloud_report
```

Comment:
why the duplication of cloud_monitor.cloud_monitor ?

Reply:
fixed